packages/kbn-es-archiver/src/lib/docs/index_doc_records_stream.ts

```diff
@@ -32,6 +32,7 @@ export function createIndexDocRecordsStream(
     await client.helpers.bulk(
       {
         retries: 5,
+        concurrency: 4,
```

Comment:
@wayneseymour can you add a short comment regarding `concurrency` number decision, so in the future we can quickly understand why it was set to 4.